src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -206,6 +207,16 @@ case class AppTerminatedEvent(
   eventType: String = "app_terminated_event",
   timestamp: String = Timestamp.now().toString) extends MarathonEvent
 
+case class UnhealthyTaskKillEvent(
+  appId: PathId,
+  taskId: Task.Id,
+  version: Timestamp,
+  health: Health,
```

Comment:
Why the complete health object? 
This event can only be thrown if there were consecutive failures, that are reported via `FailedHealthCheck`.